.travis.yml

```diff
@@ -37,11 +37,15 @@ matrix:
         - env: ASTROPY_VERSION=development
 
         # Check for sphinx doc build warnings.
-        - env: SETUP_CMD='build_sphinx -w'
+        - env: SETUP_CMD='build_sphinx -w' PIP_DEPENDENCIES='pyfftw'
```

Comment:
I think you can just use `CONDA_DEPENDENCIES='pyfftw'` and then you don't need the extra stuff below

Reply:
I specifically need to extract pyfftw from conda-forge, so I'm not sure if I can write something like `CONDA_DEPENDENCIES='-c conda-forge pyfftw'`.

Reply:
My point was that `conda` may just have packaged `pyfftw` themselves, and that that version will work. Worth trying...

Reply:
Well, that clearly didn't work! So, I guess better remove that change...